### PR TITLE
Created require_sub_types table models and migrations

### PR DESCRIPTION
**Pull request by melvinjim**

### CODE REVIEW

**Comment by melvinjim** [comment]
I am going to close this pr since I did not pay attention to the table that had to take the foreign key from another table so I am going to wait for @jolivo07  pr to be merged

File: app/models/requirement.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package models
+
+import (
+	"time"
+
+	"github.com/gofrs/uuid"
+)
+
+type Requirements struct {
+	ID                uuid.UUID `json:"id" db:"id"`
```
Comment (rdiazrovira) [inline app/models/requirement.go:10]: Ok

File: app/models/requirement.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+	RequirementTypeID uuid.UUID `json:"requirement_type_id" db:"requirement_type_id"`
+	Name              string    `json:"name" db:"name"`
+	CreatedAt         time.Time `json:"created_at" db:"created_at"`
```
Comment (jolivo07) [inline app/models/requirement.go:13]: ok

File: app/models/requirement.go [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package models
+
+import (
+	"time"
+
+	"github.com/gofrs/uuid"
+)
```
Comment (jolivo07) [inline app/models/requirement.go:7]: ok

**Pull request closed without merging**